File: Backend/src/main/java/com/resumebuilder/backend/dto/CertificationDTO.java
package com.resumebuilder.backend.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;
import jakarta.validation.constraints.NotBlank;
import java.time.LocalDate;
import java.util.List;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class CertificationDTO {
    private Long id;

    @NotBlank(message = "Certification title is required")
    private String title;

    private String issuer;
    private String link;
    private LocalDate date;
    private List<String> bulletPoints;
}

File: app/src/main/java/com/snehil/cvoptima/data/remote/model/CertificationDto.kt
package com.snehil.cvoptima.data.remote.model

import com.google.gson.annotations.SerializedName

data class CertificationDto(
    @SerializedName("id") val id: Long? = null,
    @SerializedName("title") val title: String,
    @SerializedName("issuer") val issuer: String? = null,
    @SerializedName("link") val link: String? = null,
    @SerializedName("date") val date: String? = null,
    @SerializedName("bulletPoints") val bulletPoints: List<String>? = null
)
